src/components/ResourceCard/dropdown.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter, Link } from 'react-router-dom';
+import Swal from 'sweetalert2';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Dropdown } from 'react-bootstrap';
+import { deleteResourceAction } from 'store/actions/resource';
```

Comment:
Add blank line on the above to separate npm package import and our custom input

Reply:
done